.circleci/config.yml

```diff
@@ -2,65 +2,271 @@ version: 2.1
 orbs:
   cypress: cypress-io/cypress@1
 jobs:
-  build-and-test:
+  preflight:
+    docker:
+      - image: circleci/python:3.7.3-node-browsers
+        environment:
+          DATABASE_URL: postgresql://root@localhost:5432/arlo-test
+          FLASK_ENV: test
+      - image: circleci/postgres:9.6.3-alpine-ram
+        environment:
+          POSTGRES_USER: root
+          POSTGRES_DB: arlo-test
+    resource_class: xlarge
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - run: sudo chown -R circleci:circleci /usr/local/lib/python3.7/site-packages
+      - restore_cache:
```

Comment:
maybe pull out the different caching steps as reusable commands? https://circleci.com/docs/2.0/reusing-config/

Reply:
Done

Reply:
WOW it looks so good with this change